docs/conf.py

```diff
@@ -110,10 +110,28 @@ def wiki_role(role, rawtext, text, lineno, inliner, options={}, content=[]):
     node = nodes.reference(rawtext, title, refuri=ref, **options)
     return [node], []
 
+packet_matcher = re.compile(r"(.*) <(.*)>")
+# Takes a string like raknet/client/replica/character/struct.CharacterSerialization and returns CharacterSerialization with .findall
+default_name_finder = re.compile(r"((?:[A-Z][^A-Z]*(?!\\))+$)")
```

Comment:
this seems a bit too complicated for the narrow description of what it claims to do. Does a simple split at slash or dot not suffice to implement this? Why is the casing of the struct name important here already?

Reply:
the struct.Name is guaranteed for all rustdoc, the folder nesting ia variable and depends on the module nesting lcdr chose

Reply:
Sounds good! My concern here was that, if in the future a struct would have a path, then this would catch it. But if thats not going to happen then there is no point to try and patch it